wiki/Tournaments/MWC/2018/7K/en.md

```diff
@@ -0,0 +1,212 @@
+osu!mania 7K World Cup 2018
+=============================
+
+![osu!mania 7K World Cup 2018](logo.png)
+
+The **osu!mania 7K World Cup 2018** (**_MWC 7K 2018_**) is a country-based osu!mania tournament hosted by the [osu! team](/wiki/People/The_Team). It is the 4th installment of the osu!mania 7K World Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| ---: | :--- |
+| Registration Phase | 2018-01-10/2018-01-24 |
+| Drawings | 2018-02-03 14:00:00 UTC |
+| Group Stage | 2018-02-10/2018-02-11 |
+| Round of 16 | 2018-02-17/2018-02-18 |
+| Quarterfinals | 2018-02-24/2018-02-25 |
+| Semifinals | 2018-03-03/2018-03-04 |
+| Finals | 2018-03-10/2018-03-11 |
+| Grand Finals | 2018-03-17/2018-03-18 |
+
+## Prizes
+
+In every world cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Placing | Prize(s) |
+| --- | :--- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge, "osu! Champion" user title for one year |
```

Comment:
shouldn't this be osu!mania Champion, or am I mistaken?

Reply:
You are correct, I actually changed that in the announcement but forgot to do so in the wiki article oops.